### PR TITLE
Request upload permission for Azure Virtual Machine Scale Set Plugin

### PR DESCRIPTION
**Pull request by chenkennt**
# Description

Request upload permission for Azure Virtual Machine Scale Set Plugin
https://github.com/jenkinsci/azure-vmss-plugin
https://issues.jenkins-ci.org/browse/HOSTING-446

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by daniel-beck** [comment]
Given recent permission grants, would it make sense to add @ArieShout from the start?

File: permissions/plugin-azure-vmss.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "azure-vmss"
+paths:
+- "org/jenkins-ci/plugins/azure-vmss"
+developers:
+- "chenkennt"
+- "azure_devops"
```
Comment (daniel-beck) [inline permissions/plugin-azure-vmss.yml]: `azure_devops` has never logged in to Artifactory. This blocks applying these permissions, see this repo's readme for details.

Comment (chenkennt) [reply permissions/plugin-azure-vmss.yml]: Just logged in.

**Comment by chenkennt** [comment]
Makes sense, added @ArieShout

**Pull request merged**